configs/samples/sla.conf.sample

```diff
@@ -1,7 +1,7 @@
 ;
 ; Configuration for Shared Line Appearances (SLA).
 ;
-; See http://wiki.asterisk.org or doc/AST.pdf for more information.
+; See https://docs.asterisk.org or doc/AST.pdf for more information.
```

Comment:
Pretty sure we don't generate `AST.pdf` anymore so you can remove the mention of it